docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst

```diff
@@ -91,7 +91,7 @@ Alternatively, you can download ``codeql.zip``, which contains the CLI for all s
       iv. Ensure the check-box labeled **Open "safe" files after downloading**.
           is unchecked.
 
-2. Extract the zip archive
+1. Extract the zip archive
```

Comment:
I don't think restructured text supports autonumbering with this syntax.  I suggest reverting this line, or using `#.` throughout the list to autonumber it.

Reply:
Since this is a paragraph headers, not sure if RST will do auto numbering. Let me try.

Reply:
Nope. :(